src/components/Footer.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PureComponent from 'react-pure-render/component';
+
+const styles = {
```

Comment:
ohhhh, i see. Yeah, b/c it gave me an error on dev tools that the variable was used b4 defined, totally forgot it's hoisted up.